doc/classes/JSONParseResult.xml

```diff
@@ -68,12 +70,24 @@
 	</methods>
 	<members>
 		<member name="error" type="int" setter="set_error" getter="get_error" enum="Error">
+			Returns error's type if JSON source not successfull parsed. See [@Global Scope].Error.
 		</member>
 		<member name="error_line" type="int" setter="set_error_line" getter="get_error_line">
+			Returns error's line if JSON source not successful parsed.
 		</member>
 		<member name="error_string" type="String" setter="set_error_string" getter="get_error_string">
+			Returns error message if JSON source not successful parsed. See [@Global Scope].Error.
```

Comment:
The error message if JSON source was not successfully parsed.
See above re: link.